lib/suspenders/adapters/heroku.rb

```diff
@@ -0,0 +1,50 @@
+module Suspenders
+  module Adapters
+    class Heroku
+      def initialize(app_builder)
+        @app_builder = app_builder
+      end
+
+      def set_heroku_remotes
+        remotes = <<-SHELL
+
+# Set up the staging and production apps.
+#{join_heroku_app('staging')}
+#{join_heroku_app('production')}
+      SHELL
+
+        @app_builder.append_file 'bin/setup', remotes
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.